hypershift-operator/controllers/nodepool/inplace.go

```diff
@@ -21,13 +27,40 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/cache"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/client/apiutil"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 	"sigs.k8s.io/controller-runtime/pkg/handler"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 	"sigs.k8s.io/controller-runtime/pkg/source"
 )
 
+const (
+	// CurrentMachineConfigAnnotationKey is used to fetch current targetConfigVersionHash
+	CurrentMachineConfigAnnotationKey = "machineconfiguration.openshift.io/currentConfig"
+	// DesiredMachineConfigAnnotationKey is used to indicate the version a node should be updating to
+	DesiredConfigAnnotationKey = "machineconfiguration.openshift.io/desiredConfig"
```

Comment:
DesiredConfigAnnotationKey -> DesiredMachineConfigAnnotationKey?

Reply:
Right I was a bit inconsistent here. The Annotation is the same as we use in the MCO, but the actual contents is not a desired MC but rather a nodepool configversionhash. Will revise